tests/plugins/test_compare_components.py

```diff
@@ -21,6 +21,9 @@
 from atomic_reactor.plugins.pre_reactor_config import (ReactorConfigPlugin,
                                                        WORKSPACE_CONF_KEY,
                                                        ReactorConfig)
+from atomic_reactor.plugins.post_compare_components import (
+    filter_components_by_name
+)
```

Comment:
single line, no parenthesis needed.

Reply:
I did it to pass PEP8 check due line lenght. I like more parenthesis than escaping. I can change it if escaping if it fits more current codestyle